tsconfig.base.json

```diff
@@ -6,10 +6,10 @@
         "after": true
       }
     ],
-    // nullish coalescing operator not supported in OverlayPlugin's Chrome M75
-    // TODO: change to ES2020 once OverlayPlugin rolls CEF past M80.
-    "target": "ES2019",
-    "module": "ES2020",
+    // Thanks to babel-loader, we don't need to set it as ES2019.
+    // But we need to set it as ES2020 to ensure mocha can run correctly.
+    "target": "ES2020",
+    "module": "ESNext",
```

Comment:
What do you get out of `ESNext`? Mostly wondering why you're adjusting this one as well.

Reply:
Currently `esnext` and `es2020` is the same, you mean better to keep it remained?

Reply:
Oh, I see.  Yeah, I was mostly wondering why you were changing it here.  I think I'd rather stick to ES2020 and then change it explicitly when we change it.

Reply:
Okay, done.